libtsuba/src/FileView.cpp

```diff
@@ -135,8 +135,7 @@ FileView::Fill(uint64_t begin, uint64_t end, bool resolve) {
       int err =
           mprotect(map_start_ + file_off, map_size, PROT_READ | PROT_WRITE);
       if (err == -1) {
-        KATANA_LOG_ERROR("mprotect: {}", std::strerror(errno));
-        return katana::ResultErrno();
+        return KATANA_ERROR(katana::ResultErrno(), "alloating buffer");
       }
```

Comment:
I think that should be allocating, not alloating.

Reply:
you sure? ;)

Reply:
Yeah, actually, I guess it is `mprotecting buffer`.  Is that how people say it?

Reply:
yeah I'll go `mprotecting`